src/containers/Tokens/TokensTable/index.js

```diff
@@ -72,15 +71,19 @@ const TokensTable = props => {
           )}
         </td>
         <td className="obligations">
-          {tokenInfo.obligations ? processBigNumber(tokenInfo.obligations, lng) : 0}
+          {tokenInfo.obligations
+            ? processBigNumber(tokenInfo.obligations, i18n.resolvedLanguage)
```

Comment:
Why is `i18n` used to replace `lng` on line 52, but `i18n.resolvedLanguages` used instead of `lng` here on line 75? 
(What did `lng` represent?)

Reply:
`lng` represents the language code.  Good catch.